code/workspaces/client-api/src/auth/permissionChecker.js

```diff
@@ -16,7 +16,7 @@ export const multiPermissionsWrapper = (permissionSuffixes, ...rest) => permissi
 function permissionCheck(requiredPermissions, { permissions }, next) {
   const grantedPermissions = permissions || [];
 
-  logger.info('Auth: checking permissions');
```

Comment:
Yeah, good call. Think I was a little too optimistic as to how useful this piece of information would be.